pkg/stub/handler.go

```diff
@@ -109,3 +113,35 @@ func (h *Handler) syncIngressUpdate(ci *ingressv1alpha1.ClusterIngress) error {
 
 	return nil
 }
+
+func (h *Handler) deleteIngress(ci *ingressv1alpha1.ClusterIngress) error {
```

Comment:
Should probably add a TODO to refactor this to use GC so we need only delete the DaemonSet